guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,18 +1,63 @@
+In the most common use cases, such as adding a security errata to a content view with incremental update, dependency solving is likely unnecessary and can cause significant delays to Content View publication.
 
-Note that resolving package dependencies can cause significant delays to Content View promotion.
-The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.
+The package dependency resolution feature does not take into account which packages are installed on your systems.
+Dependency resolution considers only packages within the repositories of the content view.
+ifeval::["{client-os-family}" == "Red Hat"]
+For example, if a content view has only AppStream in it, dependency solving will not know to pull dependent BaseOS content in at publish time.
+endif::[]
+For more information, see {ContentManagementDocURL}Limitations_to_Repository_Dependency_Resolution_content-management[Limitations to Repository Dependency Resolution] in _{ContentManagementDocTitle}_.
 
 .Resolving Package Dependencies and Filters
 
-Filters do not resolve any dependencies of the packages listed in the filters.
+Filters, by default, do not resolve any dependencies of the packages listed in the filters.
 This might require some level of testing to determine what dependencies are required.
 
 If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.
 
-If you create a content filter for security purposes, to resolve a package dependency, {Project} can add packages that you might consider insecure.
+If you create a content filter for security purposes but choose to resolve dependencies, {Project} can add packages that you might consider insecure.
+
+.Filtering and Dependency Solving Recommendations
+
+Be mindful when combining exclusion filters with dependency solving.
+ifeval::["{client-os-family}" == "Red Hat"]
+For example, take a scenario where a user wants to recreate RHEL 8.3 via content view filters and include select packages and errata from a later RHEL 8 y-release.
+The user excludes all errata after the RHEL 8.3 release date minus some select few that are needed and enables dependency solving.
+Dependency solving may pull in more RPMs than expected, which would cause the system to stray further from being a RHEL 8.3 machine.
+endif::[]
+If dependency solving pulls in too many unwanted RPMs, consider identifying by hand the core basic dependencies that the extra packages and errata need.
+
+
+ifeval::["{client-os-family}" == "Red Hat"]
+If, in the above case, the user does not need the extra errata and RPMs, then content view filtering is not needed at all.
+The RHEL 8.3 repository could be enabled and used directly from the Red Hat Repositories page.
```

Comment:
```suggestion
Instead, you can enable and use the RHEL 8.3 repository on *Content* > *Red Hat Repositories* in the {ProjectWebUI}.
```

Maybe drop the "instead"?

What do you think @adamlazik1 ?

Reply:
I'd probably word it like this:

```suggestion
You can enable and use the RHEL 8.3 repository on the *Content* > *Red Hat Repositories* page in the {ProjectWebUI}.
```

However, does upstream/orcharhino have this page too?

Reply:
Yes, this applies to all build targets (with Katello).

Reply:
I wanted to avoid using "you" since I mentioned "the user" above, so I'll simplify it a little further.